src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserService.cs

```diff
@@ -0,0 +1,17 @@
+using System.Linq;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Services
+{
+    public class UserService : IUserService
+    {
+        public string GetLoggedInUserEmail(HttpContext http)
+        {
+            var email = http.User.Claims.FirstOrDefault(x => x.Type == "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress")?.Value;
```

Comment:
Might be worth giving this a quick check, i noticed some oddities with the claims for dfe users, presumably being a bit different as it backed by proper AD?

Ill see if i can fine the place where i handled it

Reply:
Found it, 
DfE accounts return the email in this claim:

`"http://schemas.xmlsoap.org/ws/2005/05/identity/claims/upn"` instead of the emailaddress one

Reply:
Added upn as alternative email source if the email is blank, as per the current system.